packages/synapse-react-client/src/assets/themed_icons/SortDown.tsx

```diff
@@ -1,17 +1,14 @@
 import React from 'react'
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
 
-export const SortDown = () => (
-  <svg
+export const SortDown = (props: SvgIconProps) => (
+  <SvgIcon
```

Comment:
Use the `SvgIcon` React component provided by MUI so we can get all the features of MUI's [SvgIcon](https://mui.com/material-ui/api/svg-icon/), like theming and responsive sizing.